RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptions/ManageSubscriptionsViewModel.swift

```diff
@@ -264,8 +264,8 @@ private extension Array<CustomerCenterConfigData.HelpPath> {
         }
 
         return filter {
-            // if it's cancel, it cannot be a lifetime subscription
-            ($0.type != .cancel || !purchaseInformation.isLifetime) &&
+            // if it's cancel, it cannot be a lifetime subscription nor within trial period
+            ($0.type != .cancel || (!purchaseInformation.isLifetime || !purchaseInformation.isTrial)) &&
```

Comment:
Also, I don't have all the context, but why shouldn't we display the cancel button on a trial? Users can still cancel while on a trial right?

Reply:
Damn, this is not correct

Reply:
Thanks @tonidero

Reply:
Let me add one more test case for this, just to be sure